tests/unit/lms/resources/lti_launch_test.py

```diff
@@ -504,11 +388,6 @@ def pyramid_request(self, pyramid_request):
         return pyramid_request
 
 
-@pytest.fixture(autouse=True)
-def BearerTokenSchema(patch):
-    return patch("lms.resources.lti_launch.BearerTokenSchema")
-
-
 @pytest.fixture
-def bearer_token_schema(BearerTokenSchema):
-    return BearerTokenSchema.return_value
+def JSConfig(patch):
+    return patch("lms.resources.lti_launch.JSConfig")
```

Comment:
This is the only fixture not attached to the class. I'd tend towards localism if we can.

Reply:
This is patching `lti_launch.py`'s module-level `from lms.resources._js_config import JSConfig`, so we want the fixture to be a module-level fixture so that it applies to all of `lti_launch_test.py`. Of course `lti_launch_test.py` only contains one test class so it doesn't actually make a difference in this case yet. But I've seen it happen too many times that a test module contains multiple test classes and some of them are unwittingly using the real version of a dependency instead of the patched one, leading to confusing test bugs, of both the failing test and the nobody-noticed-this-test-was-passing-when-it-should-have-been-failing varieties. Usually happens when someone comes along and adds a new test class. So as a general rule just always put import patch fixture at module level.

It should also have `autouse=True` though, I'll fix that